src/pages/migrating/guide/design.mdx

```diff
@@ -155,55 +158,132 @@ the pixel/rem value that it is paired with.
+
+Fluid headings take the place of the v10 expressive heading. These headings are
+responsive and the type styles change size at different breakpoints.
+
+- `$expressive-heading-05` → `$fluid-heading-05`
+
+### Resources
+
+<Row className="resource-card-group">
+  <Column colLg={4} colMd={4} noGutterSm>
+    <ResourceCard
+      subTitle="Type token migration table"
+      href="https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md#type-tokens">
+      <MdxIcon name="github" />
+    </ResourceCard>
+  </Column>
 </Row>
 
-#### Layer model tokens
+## Color tokens <Tag type="cyan">Breaking</Tag>
 
-We have introduced two different types of color token for the layering
-models—Layering tokens and Contextual layer tokens. The two type of tokens will
-produce the same visual effect. The difference between them is a technical one
-and largely a developer concern. **In Sketch, and other design tools, designer
-will only be using the Layering tokens to design.** The layering tokens replace
-what were the `ui` color tokens in v10 and are used in a similar way.
+Existing color tokens have been renamed to better reflect their usage. In
+addition to renaming existing tokens, new tokens have been added to fill gaps in
+the color token system and fix complex layering logic.
 
-<Row>
-<Column colMd={6} colLg={6}>
+### Color token names
+
+Previously, in v10 many color tokens had numeral endings, now in v11 only
+layering tokens will have this distinction. All other tokens have been given an
+adjective descriptor in place of the number ending to help users better
+understand how a token should be used. The new naming convention is as follows:
 
-![Example of layering token (left) versus contextual layering tokens (right).](images/color-implementation-compare-1.png)
+`[element] - [role] - [order] - [state]`
 
-</Column>
-<Column colMd={6} colLg={6}>
+<Row>
+<Column colLg={12}>
 
-![Example of layering token (left) versus contextual layering tokens (right).](images/color-implementation-compare-2.png)
+![Example showing the v10 tokens compared to the v11 tokens](images/migrate-design-color-tokens.png)
 
 </Column>
 </Row>
 
+<Caption>
+  Examples showing v10 tokens on the left and v11 tokens on the right
+</Caption>
+
+### Layer model tokens <Tag type="green">New</Tag>
+
+We have introduced two new types of color token for the layering models—Layering
+tokens and Contextual layer tokens. The two types of tokens will produce the
+same visual effect. The difference between them is a technical one and largely a
+developer concern. **In Sketch, and other design tools, designer will only use
+the layering tokens to design.** The layering tokens replace what were the `ui`
+color tokens in v10 and are used in a similar way.
+
 See Carbon's
 [color overview](https://v11.carbondesignsystem.com/guidelines/color/overview/)
 guidance and
 [migrating color tokens](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/11.x-color.md)
```

Comment:
I think we could take this link out of this sentence, since we link out to it again under the "Resources" section you added.